cachito/workers/pkg_managers/pip.py

```diff
@@ -1448,6 +1477,115 @@ def _sdist_preference(sdist_pkg):
     return yanked_pref, filetype_pref
 
 
+def _download_vcs_package(requirement, pip_deps_dir, pip_raw_repo_name, nexus_auth):
+    """
+    Fetch the source for a Python package from VCS (only git is supported).
+
+    After downloading, upload this package to the Nexus raw repository and on subsequent
+    downloads, reuse the uploaded asset instead of fetching from VCS again.
+
+    :param PipRequirement requirement: VCS requirement from a requirements.txt file
+    :param Path pip_deps_dir: The deps/pip directory in a Cachito request bundle
+    :param str pip_raw_repo_name: Name of the Nexus raw repository for Pip
```

Comment:
I am wondering if this should be a param, given the raw hosted repository in the hoster Nexus instance should be a constant (probably in the config module).

Reply:
I fetch all the relevant values from config once in `download_dependencies()` and then pass them to the helper functions as params to avoid fetching them multiple times. I can change it to always fetch config values in helper functions if that would be preferrable.

Reply:
No need! This looks nice the way it is. I was just wondering if we needed the additional params since they are in fact constant values. Making the change would not reduce complexity anyway.

Reply:
Should I take the thumbs up as "keep it this way" or "change it"? :D